src/Feed.jsx

```diff
@@ -0,0 +1,6 @@
+const Feed = () => {
+return <>
+</>
+}
+
```

Comment:
I would add something to this even if it's a placeholder for now. I was waiting a few minutes waiting for articles to load before realising that I needed to click on the articles option in the drop down 😅

Reply:
Thanks, Rayhaan :smile: Aha, good point - sorry about that wait! I've added a feed of all articles for now - will include some functionality further down the line to show 'most popular' or something similar.